### PR TITLE
Email verify

### LINKED ISSUE #2: signup and confirm with mail

### PR DESCRIPTION
**Pull request by leducthai**

### CODE REVIEW

**Comment by leducthai** [comment]
@quocbang please review

File: server/internal/repositories/services/account/user.go [hunk 1]
```diff
@@ -4,46 +4,55 @@ import (
...
-		rd: rd,
+		pg:   pg,
+		rd:   rd,
+		smtp: sm,
 	}
 }
 
 // getAccount definition.
-func (s service) getAccount(ctx context.Context, userID string) (models.Account, error) {
+func (s service) GetAccount(ctx context.Context, Identifier string) (models.Account, error) {
```
[the remark this answers is not in the document]
Comment (leducthai) [reply server/internal/repositories/services/account/user.go:41]: @quocbang  for testing purposes

File: .github/workflows/run_test.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+      - name: generate swagger
+        run: |
+          swagger generate server --target swagger --name data-flow-sync --spec ../swagger.yml --principal models.Principal
+
+      - name: run
+        run: |
+          go vet ./...
+          go test -count=1 -gcflags=-l --coverprofile .testCoverage.txt ./...
+          go tool cover -func .testCoverage.txt
```
[the remark this answers is not in the document]
Comment (leducthai) [reply .github/workflows/run_test.yml:38]: @quocbang how can we do so?

File: server/config.yaml [hunk 3]
```diff
@@ -3,20 +3,25 @@ database:
...
 access-dir:
   - ./limitary-hour
   - ./station
   - ./station-group
   - ./material-resource
   - ./substitution
 mr-expiry-time: 24 # hour
-token-life-time: 10 # minute
+token-life-time: 5 # minute
 sync-api-server: "localhost:4422"
```
[the remark this answers is not in the document]
Comment (leducthai) [reply server/config.yaml:27]: solved

**Review by quocbang — APPROVED** [review]

**Review by quocbang — APPROVED** [review]

**Pull request merged**